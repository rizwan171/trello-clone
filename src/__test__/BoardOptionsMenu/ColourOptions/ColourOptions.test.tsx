import TestRenderer from "react-test-renderer";
import ColorOptions from "../../../components/BoardOptionsMenu/ColourOptions/ColourOptions";

describe("ColourOptions", () => {
  it("should render successfully", () => {
    const component = TestRenderer.create(<ColorOptions />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
