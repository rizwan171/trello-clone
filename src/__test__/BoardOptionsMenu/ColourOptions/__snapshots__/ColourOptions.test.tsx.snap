// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColourOptions should render successfully 1`] = `
<div
  className="grid grid-cols-6 gap-1"
>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
  <div
    className="text-center bg-green-700 rounded-md p-1"
  >
    1
  </div>
</div>
`;
